content/articles/2018-11-tidymodels-updates.Rmarkdown

```diff
@@ -0,0 +1,163 @@
+---
+title: tidymodels package updates
+date: 2018-11-28
+slug: tidymodels-update-nov-18
+author: Max Kuhn, Davis Vaughan, Alex Hayes
+categories: [package]
+description: >
+    A variety of updates for tidymodels packages.
+photo:
+  url: https://unsplash.com/photos/O6N9RV2rzX8
+  author: Jeremy Thomas
+---
+
+```{r setup, include = FALSE}
+library("tidymodels")
+
+set.seed(5283)
+
+options(digits = 3)
+knitr::opts_chunk$set(
+  collapse = FALSE, comment = "#>",
+  fig.width = 7,
+  fig.align = 'center',
+  fig.asp = 0.618, # 1 / phi
+  out.width = "700px"
+)
+```
+
+
+Along with the release of [`parsnip`](https://tidymodels.github.io/parsnip/) there are new versions of many `tidymodels` packages: [`recipes`](https://tidymodels.github.io/recipes/), [`yardstick`](https://tidymodels.github.io/yardstick/), [`embed`](https://tidymodels.github.io/embed/), [`tidyposterior`](https://tidymodels.github.io/tidyposterior/), and [`tidymodels`](https://github.com/tidymodels/tidymodels). 
+
+We made the conscious choice to add all of the breaking changes now instead of spreading them out over a few versions. The biggest changes are in `yardstick` and `recipes` and these are described below.
```

Comment:
comma between now and instead
Also, maybe ~several breaking changes— I'm just thinking that this sounds like there will never be breaking changes in the future.